### ISSUE TITLE
Can 2FA - "Composer packages are missing..."

### ISSUE THREAD

**Issue by numericOverflow** [phpmyadmin/phpmyadmin#16362]
### Describe the bug

I've installed composer packages, but php doesn't seem to be recognizing them, still complains that deps are missing to enable two-factor authentication

### To Reproduce

installed composer
restart apache2

```
ubuntu@xxx:/usr/share/phpmyadmin# composer require pragmarx/google2fa
Using version ^8.0 for pragmarx/google2fa
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 0 updates, 0 removals
  - Installing samyoul/u2f-php-server (v1.1.4): Loading from cache
Writing lock file
Generating autoload files
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!
ubuntu@xxx:/usr/share/phpmyadmin# composer require bacon/bacon-qr-code
Using version ^2.0 for bacon/bacon-qr-code
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Generating autoload files
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!
ubuntu@xxx:/usr/share/phpmyadmin# composer require samyoul/u2f-php-server
Using version ^1.1 for samyoul/u2f-php-server
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Generating autoload files
1 package you are using is looking for funding.
Use the `composer fund` command to find out more!
ubuntu@xxx:/usr/share/phpmyadmin# chown --recursive www-data:www-data ./vendor
ubuntu@xxx:/usr/share/phpmyadmin# service apache2 restart

```

### Expected behavior

Should be able to configure 2FA for users

### Screenshots

contents of:  _/usr/share/phpmyadmin/composer.json_
```
{
    "require": {
        "pragmarx/google2fa-qrcode": "^1.0",
        "phpmailer/phpmailer": "^6.0",
        "bacon/bacon-qr-code": "^2.0",
        "samyoul/u2f-php-server": "^1.1"
    }
}
```

```
root@xxx:/usr/share/phpmyadmin# ls -lah vendor/
total 60K
drwxr-xr-x  9 www-data www-data 4.0K Sep 16 15:31 .
drwxr-xr-x 10 www-data www-data  20K Sep 18 19:20 ..
-rw-r--r--  1 www-data www-data  178 Sep 16 15:28 autoload.php
drwxr-xr-x  3 www-data www-data 4.0K Sep 16 15:28 bacon
drwxr-xr-x  2 www-data www-data 4.0K Sep 16 15:31 composer
drwxr-xr-x  3 www-data www-data 4.0K Sep 16 15:28 dasprid
drwxr-xr-x  3 www-data www-data 4.0K Sep 16 15:28 paragonie
drwxr-xr-x  3 www-data www-data 4.0K Sep 16 15:28 phpmailer
drwxr-xr-x  4 www-data www-data 4.0K Sep 16 15:28 pragmarx
drwxr-xr-x  3 www-data www-data 4.0K Sep 16 15:31 samyoul
root@xxx:/usr/share/phpmyadmin#
```

![2020-09-18_140452](https://user-images.githubusercontent.com/3429424/93636911-572cab80-f9ba-11ea-9138-f7e9bd1253d8.jpg)

### Server configuration

- Operating system: Linux (Ubuntu 20.04.1 LTS) -->AWS server instance
- Web server: Apache/2.4.41 (Ubuntu) 
- Database version: 8.0.21-0ubuntu0.20.04.4 - (Ubuntu) 
- PHP version: 7.4 (and/or 7.2)
- phpMyAdmin version:  4.9.5deb2

**Comment by williamdes** [comment]
Hi @numericOverflow  
I am working on this issue upstream https://salsa.debian.org/phpmyadmin-team/phpmyadmin/-/issues/3#note_184344

The packages are still waiting on the Debian pool to be accepted.
For now you need to wait or do some tweaks to vendor/autoload.php
It autoloads Debian packages for php and not real composer files.

BaconQrCode will be accepted soon, I hope.

https://ftp-master.debian.org/new/baconqrcode_2.0.2-1.html

https://ftp-master.debian.org/new/dasprid-enum_1.0.2-1.html

**Comment by numericOverflow** [comment]
@williamdes  - Thanks for the quick response.

What sort of tweaks are needed to vendor/autoload.php?

**Comment by williamdes** [comment]
Hi @numericOverflow 
:)

Just adding a line to `vendor/autoload.php` like in https://salsa.debian.org/phpmyadmin-team/phpmyadmin/-/blob/master/debian/autoload.php.tpl#L2

and adding a file in the vendor folder could work like this one https://salsa.debian.org/php-team/pear/php-bacon-baconqrcode/-/blob/main/debian/autoload.php.tpl

I could be very much easy to install the following and tweak the `vendor/autoload.php` file
debs I built (rename .zip to .deb)
- [php-dasprid-enum_1.0.2-1_all.deb.zip](https://github.com/phpmyadmin/phpmyadmin/files/5274360/php-dasprid-enum_1.0.2-1_all.deb.zip)
- [php-bacon-qr-code_2.0.2-1_all.deb.zip](https://github.com/phpmyadmin/phpmyadmin/files/5274368/php-bacon-qr-code_2.0.2-1_all.deb.zip)

**Comment by williamdes** [comment]
:tada: 

![image](https://user-images.githubusercontent.com/7784660/94143942-cc8ff480-fe70-11ea-8e50-4cd81240b6f6.png)

Waiting now to be able to send this packages to buster-backports

**Comment by williamdes** [comment]
Hi @numericOverflow 
I build a working phpMyAdmin with BaconQrCode, but in fact it will not work until I make some changes here to make BaconQrCode v2 compatible.

For now https://github.com/phpmyadmin/phpmyadmin/blob/6590e8fbda633a8b8e0a2f0972c5e610db619378/composer.json#L64 is required for 4.9

**Comment by williamdes** [comment]
Duplicate of #16779
